test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+            string npmPath = GetNpmExecutablePath();
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments, int timeoutMilliseconds = 30000)
+        {
+            try
+            {
+                using (Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                })
+                {
+                    process.Start();
+                    string output = process.StandardOutput.ReadToEnd();
+                    string errorOutput = process.StandardError.ReadToEnd();
+                    process.WaitForExit(timeoutMilliseconds);
+
+                    int exitCode = process.ExitCode;
+                    if (exitCode != 0)
+                    {
+                        throw new ApplicationException($"Command exited with code {exitCode}. Error: {errorOutput}");
+                    }
+
+                    return output;
+                }
+            }
+            catch (Exception)
```

Comment:
I usually don't do that. If we want to map some particular exceptions to NpmNotFoundException then it would make sense to explicitly enumerate their classes. And eben in that case it is necessary to keep the original cause. "Muting" the generic exception class would make it hard to debug in the future

Reply:
Something like that is more acceptable?

If have exitCode !=0 then we throw ApplicationException and then rethrow it after we catch it 

otherwise, we will throw NpmNotFoundException()
```
catch (ApplicationException ex)
{
    throw ex;
}

catch (Exception)
{
    throw new NpmNotFoundException();
}
```

Reply:
> catch (Exception)

This part looks potentially dangerous to me. Again, only specific exception classes should be enumerated in the catch clause

Reply:
FIxed. Hopefully :)